<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitStashDataDialog</class>
 <widget class="QDialog" name="GitStashDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git Stash</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Message:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="messageEdit">
     <property name="toolTip">
      <string>Enter a message for the stash</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="keepCheckBox">
     <property name="text">
      <string>Keep changes in staging area</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Untracked/Ignored Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="noneRadioButton">
        <property name="toolTip">
         <string>Select to not stash untracked or ignored files</string>
        </property>
        <property name="text">
         <string>Don't stash untracked or ignored files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="untrackedRadioButton">
        <property name="toolTip">
         <string>Select to stash untracked files</string>
        </property>
        <property name="text">
         <string>Stash untracked files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="allRadioButton">
        <property name="toolTip">
         <string>Select to stash untracked and ignored files</string>
        </property>
        <property name="text">
         <string>Stash untracked and ignored files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>messageEdit</tabstop>
  <tabstop>keepCheckBox</tabstop>
  <tabstop>noneRadioButton</tabstop>
  <tabstop>untrackedRadioButton</tabstop>
  <tabstop>allRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GitStashDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GitStashDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
